localstack/testing/snapshots/report.py

```diff
@@ -87,16 +100,25 @@ def _render_path_part(part):
                     f"[replace](~)[/replace] {change_path} {expected!r} → {actual!r} ... (expected → actual)",
                 )
             ]
+        elif c.report_type == "type_changes":
+            return [
+                (
+                    change_path,
+                    f"[replace](~)[/replace] {change_path} {expected!r} (type: {type(expected)}) → {actual!r} (type: {type(actual)})... (expected → actual)",
+                )
+            ]
         else:
```

Comment:
Should we also return something here in the `else` part to make the issue more prominent? If we encounter another yet unknown `report_type`, it would just log the warning - which could be overlooked.